plan/planbuilder.go

```diff
@@ -452,11 +452,17 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) Plan {
 			b.err = errors.Trace(err)
 			return nil
 		}
+		solver := &aggPushDownSolver{
+			ctx:   b.ctx,
+			alloc: b.allocator,
+		}
+		solver.aggPushDown(logic)
```

Comment:
Can we call Optimize in this fucntion, so we don't need to maintain two copy of similar code?